### ISSUE TITLE
Syntax highlighting of package.json after upgrading to 1.38.0 is distracting and inconsistent

### ISSUE THREAD

**Issue by kaiyoma** [atom/language-json#75]
After upgrading from 1.37.1 to 1.38.0, syntax highlighting of package.json is distracting and inconsistent:

![atom-package-json](https://user-images.githubusercontent.com/10719780/59313858-7725d900-8c67-11e9-8653-748ce062c19d.png)

All this blue underline styling was not present before and makes the file harder to read. Also, notice the inconsistencies: most of the keys get this treatment, but not all.

### Versions

OS: Windows 10

```
$ atom --version

Atom    : 1.38.0
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

```
$ apm --version
internal/child_process.js:323
    throw errnoException(err, 'spawn');
    ^

Error: spawn UNKNOWN
    at _errnoException (util.js:1024:11)
    at ChildProcess.spawn (internal/child_process.js:323:11)
    at exports.spawn (child_process.js:502:9)
    at EventEmitter.<anonymous> (...\atom\app-1.38.0\resources\app\apm\lib\apm-cli.js:169:17)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by Aerijo** [comment]
Seems to be caused by these characters `etojcb ` (including the space)

**Comment by c22** [comment]
Is there a workaround for this? Some way of forcing atom 1.38.0 to use language-json@1.0.4?

I have tried "apm install language-json" but I get errors. The language-json@1.0.2 highlighter is almost unusable when working with large JSON files.

**Comment by c22** [comment]
I've downgraded to atom 1.37.0 and have usable highlighting again. It would be nice if there was a way to decouple the built-in language plugin versions like the good old days of this "hackable" text editor.

**Comment by mbotmcc** [comment]
> I've downgraded to atom 1.37.0 and have usable highlighting again. It would be nice if there was a way to decouple the built-in language plugin versions like the good old days of this "hackable" text editor.

haha, nothing hackable about it. Dumpted atom for vscode. Was going to happen eventually

**Comment by kevteljeur** [comment]
`apm install language-json@1.0.4` worked for me. That's like pulling a large thorn out of my foot.

**Comment by mbotmcc** [comment]
I've tried that but it kept saying that the package is already installed and I can uninstall it. When I try to uninstall it, it says the package is not installed and there's wall of errors.....

**Comment by Aerijo** [comment]
@mbotmcc Is there a folder called `language-json` in the `~/.atom/packages` folder? If you or @c22 have an issue, you can raise one with the template filled out. It is entirely possible to install `language-json`, or any other core package, through the apm command.

**Comment by rafeca** [comment]
Hey folks! We've just released Atom v1.38.1 with the fix from @Aerijo, so syntax highlighting for JSON files should work fine again out of the box
